packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol

```diff
@@ -13,50 +13,58 @@ import { SuperchainConfig } from "src/L1/SuperchainConfig.sol";
 ///         for sending and receiving data on the L1 side. Users are encouraged to use this
 ///         interface instead of interacting with lower-level contracts directly.
 contract L1CrossDomainMessenger is CrossDomainMessenger, ISemver {
-    /// @notice Address of the OptimismPortal. This will be removed in the
-    ///         future, use `portal` instead.
-    /// @custom:legacy
-    OptimismPortal public immutable PORTAL;
-
-    /// @notice Address of the SuperchainConfig contract.
+    /// @notice Contract of the SuperchainConfig.
     SuperchainConfig public superchainConfig;
 
+    /// @notice Contract of the OptimismPortal.
+    /// @custom:network-specific
+    OptimismPortal public portal;
+
     /// @notice Semantic version.
-    /// @custom:semver 2.2.0
-    string public constant version = "2.2.0";
+    /// @custom:semver 2.3.0
+    string public constant version = "2.3.0";
```

Comment:
@tynes I'm assuming that you discussed whether or not this is a change to the security model and decided no?

Reply:
We did talk about this, it should be fine to be a minor version bump everywhere